README.md

```diff
@@ -1,9 +1,10 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Leaderboard Project
 
+# LEADERBOARD Project
 
-> "Leaderboard" is a tool that helps to put the scores of leaders on the board.
+
+> "LeaderBoard" is a project to show scores of the leaders. In first part a wireframe of the project is constructed.
 
```

Comment:
- [ ] In your README file, in the `Getting started` section, you have not provided more information about how **to clone and run your project locally.** You have not included the **information about the API your app is consuming** and other information. Please try to add all of the there in that section. This will make your README look more professional 😎

Reply:
The README.md file is corrected following the review.